src/common/services/api/cart.service.spec.js

```diff
@@ -64,10 +66,11 @@ describe('cart service', () => {
       self.$httpBackend.flush()
     })
 
+    // To fetch product-code, offer resource is used and added it in zoom parameter.
```

Comment:
FYI, these comments would have been great as commit messages for separate commits.

Reply:
Sure we will follow this approach for the next commits.